spring-boot-test-autoconfigure/pom.xml

```diff
@@ -212,5 +217,17 @@
 			<artifactId>de.flapdoodle.embed.mongo</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.neo4j</groupId>
+			<artifactId>neo4j-ogm-embedded-driver</artifactId>
+			<version>${neo4j-ogm.version}</version>
+			<scope>test</scope>
```

Comment:
@eddumelendez we won't add any direct dependency to this artifact (its license is incompatible with ASLv2). Please rewrite the PR without the embedded driver (you can see how the auto-configuration for neo4j managed this).